<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDialog_fileHeader</class>
 <widget class="QDialog" name="UDialog_fileHeader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edition du fichier</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Option du fichier</string>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>341</width>
         <height>362</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Author</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_author"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_title"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Artist</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_artist"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Mp3</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_mp3"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Cover</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_cover"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Background</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit_background"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Vidéo</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lineEdit_video"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Vidéo gap</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="lineEdit_videogap"/>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Gap</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QLineEdit" name="lineEdit_gap"/>
          </item>
          <item row="13" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>BMP</string>
            </property>
           </widget>
          </item>
          <item row="13" column="1">
           <widget class="QLineEdit" name="lineEdit_bmp"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Edition</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="lineEdit_edition"/>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Language</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLineEdit" name="lineEdit_language"/>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Genre</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLineEdit" name="lineEdit_genre"/>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Year</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QLineEdit" name="lineEdit_year"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="midLineWidth">
           <number>5</number>
          </property>
          <widget class="QPushButton" name="pushButton_mp3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>77</y>
             <width>30</width>
             <height>23</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_cover">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>104</y>
             <width>30</width>
             <height>23</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_background">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>130</y>
             <width>30</width>
             <height>23</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_video">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>156</y>
             <width>30</width>
             <height>23</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UDialog_fileHeader</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UDialog_fileHeader</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
